### PR TITLE
Chagay

### PR DESCRIPTION
**Pull request by aviram804**
change sign in flow need to clean all unused files.

### CODE REVIEW

**Comment by chagayh** [comment]
Changed sign in flow.
Changed login activity and xml completely. Checked for potential bugs.

File: .idea/gradle.xml [hunk 1]
```diff
@@ -1,15 +1,19 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <project version="4">
+  <component name="GradleMigrationSettings" migrationVersion="1" />
   <component name="GradleSettings">
     <option name="linkedExternalProjectsSettings">
       <GradleProjectSettings>
-        <compositeConfiguration>
-          <compositeBuild compositeDefinitionSource="SCRIPT" />
-        </compositeConfiguration>
```
Comment (aviram804) [inline .idea/gradle.xml]: ok

File: .idea/gradle.xml [hunk 2]
```diff
@@ -1,15 +1,19 @@
...
         <option name="distributionType" value="DEFAULT_WRAPPED" />
         <option name="externalProjectPath" value="$PROJECT_DIR$" />
+        <option name="modules">
+          <set>
+            <option value="$PROJECT_DIR$" />
+            <option value="$PROJECT_DIR$/app" />
+          </set>
+        </option>
         <option name="resolveModulePerSourceSet" value="false" />
-        <option name="testRunner" value="PLATFORM" />
```
Comment (aviram804) [inline .idea/gradle.xml]: ok

File: app/build.gradle [hunk 3]
```diff
@@ -31,7 +31,6 @@ dependencies {
     implementation 'androidx.constraintlayout:constraintlayout:1.1.3'
     implementation 'com.google.firebase:firebase-firestore:21.3.1'
     implementation 'com.google.firebase:firebase-analytics:17.2.1'
-    implementation 'com.firebaseui:firebase-ui-auth:6.0.2'
```
Comment (aviram804) [inline app/build.gradle]: looks ok

File: app/build.gradle [hunk 4]
```diff
@@ -44,13 +43,14 @@ dependencies {
     testImplementation 'junit:junit:4.12'
     androidTestImplementation 'androidx.test.ext:junit:1.1.1'
     androidTestImplementation 'androidx.test.espresso:espresso-core:3.2.0'
-    implementation 'com.google.android.gms:play-services-auth:17.0.0'
-    implementation 'com.google.firebase:firebase-core:17.2.1'
-    implementation 'com.google.firebase:firebase-auth:19.2.0'
```
Comment (aviram804) [inline app/build.gradle]: ok

File: app/src/main/java/com/example/ciy/BottomNavigationBar.java [hunk 5]
```diff
@@ -226,11 +230,21 @@ private boolean showSignOutDialog() {
                 .setPositiveButton("sign out", new DialogInterface.OnClickListener() {
                     public void onClick(DialogInterface dialog, int id) {
                         Toast.makeText(BottomNavigationBar.this, "logOut", Toast.LENGTH_SHORT).show(); //TODO SHANI
-                        FirebaseAuth firebaseAuth = FirebaseAuth.getInstance();
-                        firebaseAuth.signOut();
-                        Intent intent = new Intent(getBaseContext(), LoginActivity.class);
-                        startActivity(intent);
-                        finish();
```
Comment (aviram804) [inline app/src/main/java/com/example/ciy/BottomNavigationBar.java]: mabe consider changing of FirebaseAuth to FirebaseAuthic becouse one can think the shortcat is FirebaseAuthor.

File: app/src/main/java/com/example/ciy/LoginActivity.java [hunk 6]
```diff
@@ -1,115 +1,71 @@
...
 import android.widget.Toast;
-
 import androidx.annotation.NonNull;
-
-import com.google.android.gms.auth.api.signin.GoogleSignIn;
-import com.google.android.gms.auth.api.signin.GoogleSignInAccount;
-import com.google.android.gms.auth.api.signin.GoogleSignInClient;
-import com.google.android.gms.auth.api.signin.GoogleSignInOptions;
-import com.google.android.gms.common.api.ApiException;
-import com.google.android.gms.tasks.OnCompleteListener;
```
Comment (aviram804) [inline app/src/main/java/com/example/ciy/LoginActivity.java]: cool

File: app/src/main/java/com/example/ciy/LoginActivity.java [hunk 7]
```diff
@@ -1,115 +1,71 @@
...
-import com.google.firebase.auth.AuthResult;
 import com.google.firebase.auth.FirebaseAuth;
 import com.google.firebase.auth.FirebaseUser;
-import com.google.firebase.auth.GoogleAuthProvider;
 import com.google.firebase.firestore.CollectionReference;
 import com.google.firebase.firestore.DocumentReference;
 import com.google.firebase.firestore.DocumentSnapshot;
 import com.google.firebase.firestore.FirebaseFirestore;
 import com.google.firebase.firestore.SetOptions;
-
```
Comment (aviram804) [inline app/src/main/java/com/example/ciy/LoginActivity.java]: cool

File: app/src/main/java/com/example/ciy/LoginActivity.java [hunk 8]
```diff
@@ -1,115 +1,71 @@
...
 import java.util.HashMap;
+import java.util.List;
 
-public class LoginActivity extends BaseSignIn implements View.OnClickListener {
-    private static final String TAG_1 = "EmailPassword";
-    private static final String TAG = "GoogleActivity";
-
-    private EditText mailInput;
-    private EditText passwordInput;
-
```
Comment (aviram804) [inline app/src/main/java/com/example/ciy/LoginActivity.java]: tag and tag_1 should be more informatical

File: app/src/main/java/com/example/ciy/LoginActivity.java [hunk 9]
```diff
@@ -1,115 +1,71 @@
...
+public class LoginActivity extends BaseSignIn {
 
-    private FirebaseAuth firebaseAuth;
-
-    private GoogleSignInClient mGoogleSignInClient;
-    private static final int RC_SIGN_IN = 9001;
```
Comment (aviram804) [inline app/src/main/java/com/example/ciy/LoginActivity.java]: good

File: app/src/main/java/com/example/ciy/LoginActivity.java [hunk 10]
```diff
@@ -1,115 +1,71 @@
...
-        setProgressBar(R.id.progressBar);
-        findViewById(R.id.signInGoogleButton).setOnClickListener(this);
-
-        // Configure Google Sign In
-        GoogleSignInOptions gso = new GoogleSignInOptions.Builder(GoogleSignInOptions.DEFAULT_SIGN_IN)
-                .requestIdToken(getString(R.string.default_web_client_id))
-                .requestEmail()
-                .build();
 
-        mGoogleSignInClient = GoogleSignIn.getClient(this, gso);
```
Comment (aviram804) [inline app/src/main/java/com/example/ciy/LoginActivity.java]: great

File: app/src/main/java/com/example/ciy/LoginActivity.java [hunk 11]
```diff
@@ -1,115 +1,71 @@
...
-//                            updateUI(null);       // TODO
-                        }
-
-                        // TODO add status bar
-//                        if (!task.isSuccessful()) {
-//                            mStatusTextView.setText(R.string.auth_failed);
-//                        }
-                        hideProgressBar();
-                    }
-                });
```
Comment (aviram804) [inline app/src/main/java/com/example/ciy/LoginActivity.java]: good. dont forget to do what need to do

File: app/src/main/java/com/example/ciy/LoginActivity.java [hunk 12]
```diff
@@ -131,88 +87,21 @@ public void onFailure(@NonNull Exception e) {
...
-    public void goToSignIn(View view) {
-        startActivity(new Intent(this, SignInActivity.class));
-    }
-
-    public void goToForgetPassword(View view) {
-        startActivity(new Intent(this, ForgotPasswordActivity.class));
-
-    }
-
-
```
Comment (aviram804) [inline app/src/main/java/com/example/ciy/LoginActivity.java]: please clean debug issue

File: app/src/main/java/com/example/ciy/LoginActivity.java [hunk 13]
```diff
@@ -131,88 +87,21 @@ public void onFailure(@NonNull Exception e) {
...
-                            finish();
-                        } else {
-                            // If sign in fails, display a message to the user.
-                            Log.w(TAG, "signInWithCredential:failure", task.getException());
-                            Snackbar.make(findViewById(R.id.signInActivity), "Authentication Failed.", Snackbar.LENGTH_SHORT).show();
-                        }
-                        hideProgressBar();
-                    }
-                });
-    }
```
Comment (aviram804) [inline app/src/main/java/com/example/ciy/LoginActivity.java]: very big function consider cut it to more pices

File: app/src/main/res/layout/activity_login.xml [hunk 14]
```diff
@@ -1,166 +1,9 @@
...
-        android:shadowRadius="2"
-        android:text="Login"
-        android:textColor="@color/white_smoke"
-        android:textSize="70dp"
-        app:layout_constraintEnd_toEndOf="parent"
-        app:layout_constraintStart_toStartOf="parent"
-        app:layout_constraintTop_toBottomOf="@+id/progressBar" />
-
-
-
```
Comment (aviram804) [inline app/src/main/res/layout/activity_login.xml]: -ids should be more informatical please cleat irellevant lines (90-100)

File: app/src/main/java/com/example/ciy/BottomNavigationBar.java (hunk 5 above)
Comment (aviram804) [reply app/src/main/java/com/example/ciy/BottomNavigationBar.java]: hofman will deal with that later

File: app/src/main/java/com/example/ciy/LoginActivity.java (hunk 11 above)
Comment (aviram804) [reply app/src/main/java/com/example/ciy/LoginActivity.java]: hofman will deal with that later

File: app/src/main/java/com/example/ciy/LoginActivity.java (hunk 13 above)
Comment (aviram804) [reply app/src/main/java/com/example/ciy/LoginActivity.java]: hofman will deal with that later

File: app/src/main/res/layout/activity_login.xml (hunk 14 above)
Comment (aviram804) [reply app/src/main/res/layout/activity_login.xml]: hofman will deal with that later

File: app/src/main/java/com/example/ciy/LoginActivity.java (hunk 12 above)
Comment (aviram804) [reply app/src/main/java/com/example/ciy/LoginActivity.java]: hofman will deal with that later

**Comment by aviram804** [comment]
mere new sign in flow

**Pull request merged**